client/settings/settings-manager/index.js

```diff
@@ -110,9 +111,23 @@ const DepositsDescription = () => {
 };
 
 const FraudProtectionDescription = () => {
+	const { isLoading } = useSettings();
```

Comment:
I think this logic would fit better being part of the `SettingsManager` instead of this component. What do you think?